selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -323,10 +325,12 @@ public void waitAllInvisibilityBy(List<By> elementsLocators, int timeout) {
   }
 
   /**
-   * Waits until {@link WebElement} with provided locator {@link By} be attached to DOM it does not
-   * mean that element is visible
+   * Waits during {@code timeout} until {@link WebElement} with provided {@code elementLocator} be
```

Comment:
`be attached` > `is attached`
> In clauses with time words like when, after, until we often use the present tense forms to talk about the future.
https://learnenglish.britishcouncil.org/en/english-grammar/verbs/verbs-time-clauses-and-if-clauses